install/firmware.csv

```diff
@@ -496,6 +496,7 @@ Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40034,5.45,2020-09-28T08:36:22Z,9999-
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40069,5.45,2020-06-23T01:06:32Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,5.43,2020-02-12T03:08:38Z,2020-09-28T04:50:04Z,
 Trimble Navigation Ltd.,TRIMBLE ALLOY,5946R40072,5.45,2020-09-28T04:50:05Z,9999-01-01T00:00:00Z,
+Trimble Navigation Ltd.,TRIMBLE ALLOY,5951R40056,5.45,2021-06-01T21:46:08Z,9999-01-01T00:00:00Z,
```

Comment:
can you please update the sn to `15946R40026`?

Reply:
Done